crates/testlang/outputs/cargo/tests/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "testlang_cargo_tests"
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+publish = false
+
+[build-dependencies]
```

Comment:
I wonder if `[build-dependencies]` is needed here as well, given there is no `build.rs`?

Reply:
Done